saga-web/src/main/resources/saga-frontend/package.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "saga-frontend",
+  "version": "2.3.0",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/akveo/ngx-admin.git"
+  },
+  "bugs": {
+    "url": "https://github.com/akveo/ngx-admin/issues"
+  },
+  "browserslist": [
+    "> 1%",
+    "last 2 versions",
+    "IE 11"
+  ],
+  "scripts": {
+    "ng": "ng",
+    "conventional-changelog": "conventional-changelog",
+    "start": "ng serve --port 5200",
+    "build": "ng build",
+    "build:prod": "npm run build -- --prod --aot",
+    "test": "ng test",
+    "test:coverage": "rimraf coverage && npm run test -- --code-coverage",
+    "lint": "ng lint",
+    "lint:fix": "ng lint saga-frontend --fix",
+    "lint:styles": "stylelint ./src/**/*.scss",
+    "lint:ci": "npm run lint && npm run lint:styles",
+    "pree2e": "webdriver-manager update --standalone false --gecko false",
+    "e2e": "ng e2e",
+    "docs": "compodoc -p src/tsconfig.app.json -d docs",
+    "docs:serve": "compodoc -p src/tsconfig.app.json -d docs -s",
+    "prepush": "npm run lint:ci",
+    "release:changelog": "npm run conventional-changelog -- -p angular -i CHANGELOG.md -s"
+  },
+  "dependencies": {
```

Comment:
I guess some of this might not be required after trimming down the UI to minimal, so please double check this list

Reply:
only the dependencies that are required are kept. remaining have been removed.